tests/test_install.py

```
@@ -339,6 +348,104 @@ def test_build_and_plan_prompts_define_their_boundaries() -> None:
+    assert skill.read_text(encoding="utf-8") == "operator-owned TDD policy\n"
+    assert not any("TDD skill" in str(item.message) for item in caught)
+
+
+def test_enabled_tdd_operator_skill_conflict_precedes_mutation(tmp_path: Path) -> None:
+    write_tdd_config(tmp_path, enabled=True)
+    skill = tmp_path / ".opencode/skills/develop-tdd/SKILL.md"
+    skill.parent.mkdir(parents=True)
+    skill.write_text("operator-owned TDD policy\n", encoding="utf-8")
+    before = _tree_manifest(tmp_path)
+
+    with pytest.raises(FileExistsError, match="develop-tdd/SKILL.md"):
```

Ruff (0.16.1)

[warning] Pattern passed to `match=` contains metacharacters but is neither escaped nor raw

(RUF043)